src/components/theme-selector/themes.api.ts

```diff
@@ -1,5 +1,5 @@
-import { themesApiFixture } from '../../../../test/fixtures/themes.api.fixture'
-import { ConfigModel } from '../../../services/themes/themes.model'
+import { themesApiFixture } from '../../../test/fixtures/themes.api.fixture'
+import { ConfigModel } from '../../services/themes/themes.model'
 
 class ThemesApi {
```

Comment:
It currently only serves to use the fixture, but I would also move it into `services/themes`